dist/cms.js

```diff
@@ -35,10 +35,52 @@ function getWordPressEmbedBlockCounts() {
   }
 }
 
+/**
+ * Obtains data about scripts that WordPress prints on the page.
+ *
+ * @returns {{}[]}
+ */
+function getWordPressScripts() {
+  const entries = [];
+
+  /**
+   * Checks whether the provided element is an inline script.
+   *
+   * @param {?Element} element Element to examine. May be null.
+   * @return {?number} Size of the inline script or null if the element isn't an inline script.
+   */
```

Comment:
This isn't an accurate comment.

```suggestion
  /**
   * Returns the number of characters in an inline script.
   * Returns null for non-inlined scripts
   *
   * @param {?Element} element Element to examine. May be null.
   */
```

Reply:
Good catch. I wrote this initially when it was just checking if it was present. Thank you.